package/MDAnalysis/topology/LAMMPSParser.py

```diff
@@ -102,69 +116,8 @@
 
 logger = logging.getLogger("MDAnalysis.topology.LAMMPS")
 
-
-# Sections will all start with one of these words
-# and run until the next section title
-SECTIONS = set([
-    'Atoms',  # Molecular topology sections
-    'Velocities',
-    'Masses',
-    'Ellipsoids',
-    'Lines',
-    'Triangles',
-    'Bodies',
-    'Bonds',  # Forcefield sections
-    'Angles',
-    'Dihedrals',
-    'Impropers',
-    'Pair',
-    'Pair LJCoeffs',
-    'Bond Coeffs',
-    'Angle Coeffs',
-    'Dihedral Coeffs',
-    'Improper Coeffs',
-    'BondBond Coeffs',  # Class 2 FF sections
-    'BondAngle Coeffs',
-    'MiddleBondTorsion Coeffs',
-    'EndBondTorsion Coeffs',
-    'AngleTorsion Coeffs',
-    'AngleAngleTorsion Coeffs',
-    'BondBond13 Coeffs',
-    'AngleAngle Coeffs',
-])
-# We usually check by splitting around whitespace, so check
-# if any SECTION keywords will trip up on this
-# and add them
-for val in list(SECTIONS):
-    if len(val.split()) > 1:
-        SECTIONS.add(val.split()[0])
-
-
-HEADERS = set([
-    'atoms',
-    'bonds',
-    'angles',
-    'dihedrals',
-    'impropers',
-    'atom types',
-    'bond types',
-    'angle types',
-    'dihedral types',
-    'improper types',
-    'extra bond per atom',
-    'extra angle per atom',
-    'extra dihedral per atom',
-    'extra improper per atom',
-    'extra special per atom',
-    'ellipsoids',
-    'lines',
-    'triangles',
-    'bodies',
-    'xlo xhi',
-    'ylo yhi',
-    'zlo zhi',
-    'xy xz yz',
-])
+# NOTE: The SECTIONS, HEADERS, and ATOM_STYLES are only used in DATAParser, so
```

Comment:
Keeping this comment here doesn't really make sense if we have removed the data.